app-emacs/treemacs/treemacs-2.9.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS="26.1"
+
+inherit elisp
+
+DESCRIPTION="A tree layout file explorer for Emacs"
+HOMEPAGE="https://github.com/Alexander-Miller/treemacs"
+SRC_URI="https://github.com/Alexander-Miller/treemacs/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="evil magit icons-dired" # some flags need packages not yet in gentoo
```

Comment:
Which flags? Flags you want to add but aren't here? Or...?

I'd maybe do this instead:
```suggestion
# TODO: some possible extra flags need packages not yet in gentoo
# foo: would need app-emacs/bar
# ...
IUSE="evil magit icons-dired"
```